media/transcoder/src/test/java/com/example/transcoder/CreateJobFromAdHocTest.java

```
@@ -0,0 +1,150 @@
+    PROJECT_ID = requireEnvVar("GOOGLE_CLOUD_PROJECT");
+    PROJECT_NUMBER = requireEnvVar("GOOGLE_CLOUD_PROJECT_NUMBER");
+  }
+
+  @Before
+  public void beforeTest() throws IOException {
+    bout = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(bout));
+
+    bout = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(bout));
+
+    deleteBucket(BUCKET_NAME);
+    Storage storage = StorageOptions.newBuilder().setProjectId(PROJECT_ID).build().getService();
+    storage.create(
+        BucketInfo.newBuilder(BUCKET_NAME)
+            .setStorageClass(StorageClass.STANDARD)
+            .setLocation(LOCATION)
+            .build());
+
+    BlobId blobId = BlobId.of(BUCKET_NAME, TEST_FILE_NAME);
+    BlobInfo blobInfo = BlobInfo.newBuilder(blobId).build();
+    Path path = Paths.get(TEST_FILE_PATH);
+    storage.create(blobInfo, Files.readAllBytes(path));
+    bout.reset();
+  }
+
+  @Test
+  public void test_CreateJobFromAdHoc() throws Exception {
+    String jobName = String.format("projects/%s/locations/%s/jobs/", PROJECT_NUMBER, LOCATION);
+    try {
+      CreateJobFromAdHoc.createJobFromAdHoc(PROJECT_ID, LOCATION, INPUT_URI, OUTPUT_URI_FOR_AD_HOC);
+    } catch (GoogleJsonResponseException gjre) {
+      // Handle error
+    }
+    String output = bout.toString();
+    assertThat(output, containsString(jobName));
+    String[] arr = output.split("/");
+    JOB_ID = arr[arr.length - 1].replace("\n", "");
+    bout.reset();
+
+    Thread.sleep(60000);
+
+    try {
+      GetJobState.getJobState(PROJECT_ID, LOCATION, JOB_ID);
+    } catch (GoogleJsonResponseException gjre) {
+      // Handle error
+    }
+    output = bout.toString();
+    assertThat(output, containsString("SUCCEEDED"));
+    bout.reset();
+  }
+
+  @After
+  public void tearDown() throws IOException {
+    try {
+      DeleteJob.deleteJob(PROJECT_ID, LOCATION, JOB_ID);
+    } catch (GoogleJsonResponseException gjre) {
+      // Handle error
+    }
```

Why are you catching and ignoring?  as above, if you catch, you should do something, if you wish to ignore an exception, please say so.  Typically using the word "expected" as the variable name. https://google.github.io/styleguide/javaguide.html#s6.2-caught-exceptions

---

Removed try-catch